src/lane_keep_assist/msg/LaneStatus.msg

```diff
@@ -0,0 +1,8 @@
+int8 lane_gradient_status
+int8 lane_hls_status
+int8 lane_segmentation_status
+Lane[] gradient_lanes
```

Comment:
Is any number of Lanes allowed?

Reply:
technically yes but currently the max will only every be two, this number is described with the status messages. I'll add a lane enum so its more clear as to what each message means.

Reply:
rename gradient lanes to gradient_lane_bounds